lib/item.rb

```diff
@@ -1,14 +1,12 @@
 class Item
   attr_accessor :publish_date
-  attr_reader :id, :genre, :author, :label, :archived
 
-  def initialize(genre, author, label, date, archived: false)
+  # attr_reader :id, :genre, :author, :label
```

Comment:
Kindly remove the code that's been commented out so that we can maintain a clean codebase.